baseapp/params.go

```diff
@@ -6,10 +6,21 @@ import (
 	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
 )
 
+var InitialAppVersion uint64 = 0
```

Comment:
[nit] golangci is complaining here

Reply:
will do in a follow up, to not clutter git history here (if merged with a merged commit)